spec/requests/api/v1/tracks_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'spec_helper'
+
+describe 'API tracks' do
+  let!(:conference) { create(:conference) }
+  let!(:track1) { create(:track, program_id: conference.program.id) }
```

Comment:
This track is supposed to have the name 'Example Track', but to check that I would have to go back to the factory. You can set the name of the track here and call it 'Conference Track' or something like that, so that you can also lose the comment on line 34. The name of the variable can also be something like track_of_conference instead of track1, so that it makes more sense.